client/.eslintrc.json

```diff
@@ -14,18 +14,16 @@
         "no-underscore-dangle": "off",
         "no-plusplus": "off", // keep `var ++` for consistency
         "no-use-before-define": ["error", {"functions":  false, "classes":  false}],
+        "react/forbid-prop-types": "off", // we're passing obj/array
         "react/react-in-jsx-scope": "off",
         "react/jsx-filename-extension": "off",
         "react/jsx-props-no-spreading": "off",
-        "react/prop-types": "off", // should refactor later
-        "react/no-array-index-key": "off", // should refactor later
-        "no-nested-ternary": "off", // should refactor later
         "jsx-a11y/click-events-have-key-events": "off", // low-prio refactor
         "jsx-a11y/no-static-element-interactions": "off", // low-prio refactor
         "jsx-a11y/no-noninteractive-element-interactions": "off", // low-prio refactor
-        "react/destructuring-assignment": "off", // should refactor later
         "radix": "off", // low-prio refactor
-        "no-param-reassign": "off", // should refactor later
+        // ↓↓ in src/actions we reassign pub date to keep only year YYYY ↓↓
+        "no-param-reassign": ["error", {"props": true, "ignorePropertyModificationsFor": ["pub"]}], 
```

Comment:
Can I keep this? Or do we prefer turning it on and do refactor in API calls?
The code that's violating this rule:
```
In client/src/actions, line 24 and line 134:

    // only get the year from the date
    data.map((pub) => (pub.yearPublished = pub.yearPublished.substring(0, 4)));


```

Reply:
You can keep it if it's too hard to refactor or make a ticket to fix it up in the backend in another PR. I think the reason why this isn't done in the backend cause it just returns the doc from DB straight away in the timestamp format.